docs/setup-cluster/k8s/resource-caps.rst

```diff
@@ -0,0 +1,100 @@
+.. _k8s-resource-caps:
+
+#####################################
+ Manage Workspace Namespace Bindings
+#####################################
+
+.. note::
+
+   Managing resource quotas in Determined requires the Determined Enterprise Edition.
+
+Determined makes it easier to manage resource quotas by handling them directly, so you don't need to
+manage them separately in Kubernetes. You can manage workspace namespace bindings and resource
+quotas using either the WebUI or CLI.
+
+*******
+ WebUI
+*******
+
+#. As an admin user, either create a new workspace or edit an existing workspace.
+
+#. In the "Namespace Bindings" section, type a name for the namespace or leave it blank. If left
+   blank, the workspace will be bound to the default namespace configured in the Master
+   Configuration YAML file.
+
+#. Toggle the "Auto Create Namespace" option on or off. When enabled, you can edit the resource
+   quota directly in Determined.
+
+#. Enter a resource quota (e.g., 5). This quota is editable only when "Auto Create Namespace" is on.
+   If the quota is set in Kubernetes, it will appear here but won't be editable.
+
+When completed, save your changes to apply the namespace bindings and resource quotas.
```

Comment:
Do we want to say what the quota actually does? Like "any submitted workloads which would cause workspace resource use to exceed the defined cap will stay pending"

Reply:
Any submitted workloads that would cause workspace resource use to exceed the defined quota would stay pending (until the slot becomes available, e.g., when user kills a notebook that was using the slot).